<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Beta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mainTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ownerTab">
     <attribute name="title">
      <string>Proprietários</string>
     </attribute>
     <widget class="QTableWidget" name="ownerWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>1041</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1041</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Proprietários</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="createOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>470</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo Proprietário</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover Proprietário</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>530</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterar Dados de Prop.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="readOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Nome</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchNameLine">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>220</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>80</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>440</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>430</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="birdTab">
     <attribute name="title">
      <string>Pássaros</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1241</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Bicudo Fibra</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bicudo Fibra</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="bicudoWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshBicudoBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createBirdBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Pássaro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBirdBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Pássaro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBirdBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de Pássaro</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Curió Fibra</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Chanchão Fibra</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Coleiro</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Trinca Ferro</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Curió ?</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="chanchaoTab">
     <attribute name="title">
      <string>Competição</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
